tests/Feature/BookTest.php

```diff
@@ -114,6 +114,39 @@ public function test_getAllBooks_genre_error(): void
             ->assertInvalid(['genre']);
     }
 
+    public function test_getAllBooks_search(): void
+    {
+        Book::factory()->count(1)->create(['title' => 'enim ']);
```

Comment:
You should add a second book that won't be returned by your search. At the moment, you could actually delete the search code entirely and this test would still pass